sympy/integrals/tests/test_integrals.py

```diff
@@ -1070,17 +1070,36 @@ def test_issue_2708():
 
 def test_issue_8368():
     assert integrate(exp(-s*x)*cosh(x), (x, 0, oo)) == \
-        Piecewise((pi*Piecewise((-s/(pi*(-s**2 + 1)), Abs(s**2) < 1),
-        (1/(pi*s*(1 - 1/s**2)), Abs(s**(-2)) < 1), (meijerg(((S(1)/2,), (0, 0)),
-        ((0, S(1)/2), (0,)), polar_lift(s)**2), True)),
-        And(Abs(periodic_argument(polar_lift(s)**2, oo)) < pi,
-        cos(Abs(periodic_argument(polar_lift(s)**2, oo))/2)*sqrt(Abs(s**2)) -
-        1 > 0, Ne(s**2, 1))), (Integral(exp(-s*x)*cosh(x), (x, 0, oo)), True))
+        Piecewise(
+            (   pi*Piecewise(
```

Comment:
LOL understood

Here's how I arrived at that:

I was going for

```
function(
    subexpression1,
    subexpression2
)
```

Now for tuples, I do not have a function name, which gave me

```
(
    subexpression1,
    subexpression2
)
```

which looks silly and wastes an entire line for the merely structural opening brace. So I folded the first line back but had to keep the subexpressions aligned, which gave me

```
(   subexpression1,
    subexpression2
)
```

I'm not really happy with that myself; better proposals how to format multi-line tuples very much appreciated.